File: packages/language-typescript/spec/fixtures/sample.ts
// @ts-nocheck
/* eslint-disable */

import json from './test.json' with { type: 'json' };
//                             ^^^^ keyword.control.with.ts

type World = "world";
// <- storage.type.ts
//   ^^^^^ support.storage.other.type.ts
//           ^^^^^^^ string.quoted.double.ts


type Greeting = `hello ${World}`;
//              ^^^^^^^^^^^^^^^^ string.quoted.backtick.ts
//                     ^^^^^^^^  meta.embedded.line.interpolation.ts
//                     ^^ punctuation.section.embedded.begin.ts
//                            ^ punctuation.section.embedded.end.ts

File: packages/language-java/spec/fixtures/sample.java
// Test
// <- punctuation.definition.comment.begin
// ^^^^ comment.line

/* Test */
// <- punctuation.definition.comment.begin
// ^^^^^^ comment.block
//      ^^ punctuation.definition.comment.end

public class Test {
  public static void main(String[] args) {
    String test = """
    """;
//  ^^^ string.quoted.triple.block.java
//  ^^^ punctuation.definition.string.end
    System.out.println(test);
  }
}
